tests/Cargo.toml

```diff
@@ -4,10 +4,16 @@ version = "0.25.2"
 edition = "2021"
 
 [dev-dependencies]
+slog = { version = "2.7.0", features = ["release_max_level_trace", "max_level_trace"] }
 bollard = "0.10"
 tokio = {version = "1.16.1", features = ["rt", "macros", "process"]}
 tokio-stream = "0.1"
 futures = "0.3.13"
 port_check = "0.1.5"
 anyhow = "1.0.54"
 lazy_static = "1.4.0"
+graph-node = { path = "../node" }
+graph-core = { path = "../core" }
+graph = { path = "../graph" }
+graph-mock = { path = "../mock" }
+graph-store-postgres = { path = "../store/postgres" }
```

Comment:
Why did all these deps become necessary?

Reply:
This is for code that's not yet committed, the tests I've been working on

Reply:
Then they should be removed if they are dead deps for now.

Reply:
Then they should be removed if they are dead deps for now.